src/ca11y/index.js

```diff
@@ -4,32 +4,93 @@ import zeroPad  from './lib/zeroPad'
 import keys     from './lib/keys'
```

Comment:
I think he's suggesting that, further up in the file, the zeroPad import should be removed (it's no longer used by the library, maybe?)